src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -401,10 +440,48 @@ private string ConfigureDatabase(InitializeViewRequestParams viewParams, Databas
                         {
                             prototype.DatabaseCompatibilityLevel = compatLevelEnums[database.CompatibilityLevel];
                         }
-                        if (prototype is DatabasePrototype110 db110 && database.ContainmentType != null)
+                        if (prototype is DatabasePrototype80 db80)
                         {
-                            db110.DatabaseContainmentType = containmentTypeEnums[database.ContainmentType];
+                            if (database.DatabaseReadOnly != null)
+                            {
+                                db80.IsReadOnly = (bool)database.DatabaseReadOnly;
+                            }
+                        }
+
+                        if (prototype is DatabasePrototype90 db90)
+                        {
+                            db90.AutoUpdateStatisticsAsync = database.AutoUpdateStatisticsAsynchronously;
+                            db90.PageVerifyDisplay = database.PageVerify;
                         }
+                        if (prototype is DatabasePrototype100 db100)
+                        {
+                            db100.EncryptionEnabled = database.EncryptionEnabled;
+                        }
+                        if (prototype is DatabasePrototype110 db110)
+                        {
+                            if (database.TargetRecoveryTimeInSec != null) {
+                                db110.TargetRecoveryTime = (int)database.TargetRecoveryTimeInSec;
+                            }
+
+                            if (database.ContainmentType != null)
+                            {
+                                db110.DatabaseContainmentType = containmentTypeEnums[database.ContainmentType];
+                            }
+                        }
+                        if (prototype is DatabasePrototype160 db160)
+                        {
+                            if (database.IsLedgerDatabase != null)
+                            {
+                                db160.IsLedger = (bool)database.IsLedgerDatabase;
+                            }
+                        }
+
+                        // AutoCreateStatisticsIncremental can only be set when AutoCreateStatistics is enabled
```

Comment:
Do you need to add an if case here to account for this condition?

Reply:
We do not show any message or error when the AutoCreateStatisticsIncremental is set to true when the other one is set to false, we are just closing the window when saved and doing no updates to this property. I found these are interrelated and added a comment here. Also, wouldn't cause anything if we add a condition here!